### DISCUSSION TITLE
how to add built-in schema?

### DISCUSSION THREAD

**Discussion by cea5** [lf-edge/ekuiper#2262] [Q&A]
<img width="248" alt="image" src="https://github.com/lf-edge/ekuiper/assets/17960530/08343670-8e96-4be3-8901-7ac2249493fc">

I try to add the proto file to etc/schemas/protobuf, dosen't work.

**Comment by ngjaying** [comment]
If you manually put it into the file system, need to restart eKuiper. Use API if live load is needed.
